### PR TITLE
Naaoli development

### PR DESCRIPTION
**Pull request by naaoli**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -22,43 +28,55 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.json.simple.JSONArray;
-
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: missing blank line before this comment.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -22,43 +28,55 @@
...
     response.getWriter().println(messageList.toString());
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get Input from the Form
     String preproccessedText = request.getParameter("user-input");
-    if(preproccessedText == null) {
+
+    if(request == null) {
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:58]: missing space after `if`. This should be: `if (request == null)`

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
"Separating any reserved word, such as if, for or catch, from an open parenthesis (() that follows it on that line"

please fix this for other instances in this file (`for` and `if` primarily)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -22,43 +28,55 @@
...
       response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: should this just `return` if it's a bad request? why did you remove the return?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -22,43 +28,55 @@
...
-    for(int i = 0; i < greetings.length; i++) {
-      messageList.add(greetings[i]);
+    for(String str : greetings) {
+      commentEntity = new Entity("Comment");
+      commentEntity.setProperty("Comment", str);
+      datastore.put(commentEntity);
+      messageList.add(str);
     }
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:78]: Why do you need this line?

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
       <title>My Portfolio</title>
       <link rel="stylesheet" href="style.css">
   </head>
-  <body onload="getHelloName()">
+  <body onload="load()">
```
Comment (wli) [inline portfolio/src/main/webapp/index.html:8]: Sorry, I don't remember, was this previously changed in javascript already? I don't see a change from `getHelloName` to `load` in this PR.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (naaoli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: Just to check if the displaying of comments was working

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (naaoli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:78]: To store the comments into datastore.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (naaoli) [reply portfolio/src/main/webapp/index.html:8]: It was changed in an earlier PR but I forgot to change the function name in the html file.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (wli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:81]: Shouldn't this go before line 77? You use the `datastore` variable before you set it to something. I'm confused - does this compile?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 6]
```diff
@@ -28,37 +34,50 @@
...
     response.getWriter().println(messageList.toString());
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get Input from the Form
     String preproccessedText = request.getParameter("user-input");
-    if(preproccessedText == null) {
+
+    if (request == null) {
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: this check seems wrong - you still want to check if `preprocessedText` is null. If you want to check whether `request` is wrong, you need to check it before line 57 where you use it.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -16,15 +16,19 @@
...
   document.getElementById('comment-form').addEventListener('submit', async function(e) {
     e.preventDefault();
     const response = await fetch('/data', {
       method: 'POST'
     });
+
     const json = await response.json();
     document.getElementById('comment-container').innerText = json;
+
+    console.log(json);
```
Comment (wli) [inline portfolio/src/main/webapp/script.js:28]: remove extra console log statement

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 8]
```diff
@@ -14,10 +14,76 @@
```
Comment (wli) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:14]: week 5 changes should be in a separate pull request

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 9]
```diff
@@ -28,25 +34,33 @@
...
+    response.setContentType("application/json;");
     response.getWriter().println(messageList.toString());
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get Input from the Form
     String preproccessedText = request.getParameter("user-input");
-    if(preproccessedText == null) {
+<<<<<<< HEAD
```
Comment (wli) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:58]: you have merge markers you need to remove

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java [hunk 10]
```diff
@@ -93,4 +93,9 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
```
Comment (wli) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java:96]: please remove these two week 5 files too
